signalfx-codahale/pom.xml

```diff
@@ -61,20 +62,18 @@
       <artifactId>metrics-core</artifactId>
     </dependency>
 
+    <!-- test -->
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
-      <scope>test</scope>
```

Comment:
why remove the scope here?

Reply:
oh, does it get it inherited from the main pom?

Reply:
Yes, I removed all inherited test scopes